pynicotine/gtkgui/ui/dialogs/about.ui

```diff
@@ -202,32 +202,35 @@ Attributions
   Copyright (c) 2014-2018 Tom Wallroth
   https://github.com/devsnd/tinytag/
 </property>
-    <property name="translator-credits">English
- - mathiascode (2020)
+<property name="translator-credits">
```

Comment:
This `property` tag and the `/property` tag above it both should have indentation of 4 spaces